geograph/geograph.py

```diff
@@ -107,6 +107,16 @@ def __init__(
             **apply_buffer (bool, optional): Apply shapely buffer function to
                 the polygons after polygonising. This can fix issues with the
                 polygonisation creating invalid geometries.
+
+        Example::
+            >>> import os
+            >>> from geograph.constants import WGS84
+            >>> from geograph.tests.create_data_test import TEST_DATA_FOLDER
+            >>> gg = GeoGraph(os.path.join(TEST_DATA_FOLDER, "adjacent", "full.gpkg"), WGS84, "savepath.gz")
+            Graph successfully loaded with 15 nodes and 40 edges.
+            >>> gg.crs == WGS84
+            True
```

Comment:
Cool! I didn't know you could integrate this so nicely into dockstrings, that's really cool